histomicstk/preprocessing/color_conversion/rgb_to_hsi.py

```diff
@@ -14,5 +18,6 @@ def rgb_to_hsi(im):
     hues = (np.arctan2(3**0.5 * (im[1] - im[2]),
                        2 * im[0] - im[1] - im[2]) / (2 * np.pi)) % 1
     intensities = im.mean(0)
-    saturations = np.where(intensities, 1 - im.min(0) / intensities, 0)
+    saturations = np.where(
+        intensities, 1 - im.min(0) / (intensities + 1e-10), 0)
```

Comment:
Won't intensities only be zero if min is also zero?  Rather than add a bit of noise to all conditions, would it be better to use `np.maximum(intensities, 1e-10)`?